src/h5player/comment.js

```diff
@@ -1,5 +1,5 @@
 // ==UserScript==
-// @name         HTML5视频播放器增强脚本
```

Comment:
There may be side effects of changing the official name. Possibly TM or GreasyFork might think it's a new script, rather than a change to the existing one. 
https://greasyfork.org/en/discussions/greasyfork/75556-when-a-script-name-is-changed-greasyfork-doesn-t-recognize-the-old-script
This needs to be well tested. Changing description and other things should be safe and could be first step.